.ci/pytorch/test.sh

```diff
@@ -1313,10 +1313,10 @@ EOF
 
   # Step 2. Make sure that the public API test "test_correct_module_names" fails when an existing
   # file is modified to introduce an invalid public API function.
-  EXISTING_FILEPATH="${TORCH_INSTALL_DIR}/nn/parameter.py"
+  EXISTING_FILEPATH="${TORCH_INSTALL_DIR}/cuda/streams.py"
```

Comment:
This PR added __all__ to nn/parameter.py which makes this test not fail in the expected manner

Reply:
Done!